soroban-env-common/src/meta.rs

```diff
@@ -56,5 +56,5 @@
 // implementations over a long period of time.
 
 soroban_env_macros::generate_env_meta_consts!(
-    interface_version: 23,
+    interface_version: 24,
```

Comment:
I don't think this needs an interface bump. Nothing in env.rs is changing

Reply:
I undid the version bump.